modules/react/menu/stories/examples/ContextMenu.tsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 
 import {Menu} from '@workday/canvas-kit-react/menu';
+import {BodyText} from '@workday/canvas-kit-react/text';
 
 export const ContextMenu = () => {
   const [selected, setSelected] = React.useState('');
   return (
     <Menu onSelect={data => setSelected(data.id)}>
-      <Menu.TargetContext>Open Menu</Menu.TargetContext>
+      <Menu.TargetContext>Right-click to Open Menu</Menu.TargetContext>
```

Comment:
Is it customary for a context menu to trigger off of a button? Feels odd to me.

In any case, I added "Right-click" to the button label to make it clearer how to trigger the context menu.